modules/apps/knowledge-base/knowledge-base-test/src/testIntegration/java/com/liferay/knowledge/base/service/test/KBArticleLocalServiceTest.java

```diff
@@ -1118,6 +1119,17 @@ public void testExpireKBArticleWithLockByPreviousUser() throws Exception {
 				_serviceContext));
 	}
 
+	@Test
+	public void testFetchPersistedModelByPrimaryKey() throws Exception {
```

Comment:
public void testFetchPersistedModel() throws Exception {

Reply:
@AliciaGarciaGarcia, why not go with resourcePrimKey like how you mentioned in journal?

Reply:
Changed, but as testFetchPersistedModelByResourcePrimKey to remark that we are testing the request by ResourcePrimKey, not the default by Id.

Reply:
@jkappler 
As explained to Manu the method it's checking is testFetchPersistedModel but I did not know that the method is precissaly different because now check by ByResourcePrimKey not by id.

Ans on Journal we are checking 2 diferent ways to test the testFetchPersistedModel and I thought that was better taht way

I talked with him and gave me reasons and find a middle point :)